build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
 
     implementation "org.slf4j:slf4j-log4j12"
     implementation "org.slf4j:slf4j-api"
+    implementation "com.exactpro.th2:grpc-common:3.5.0"
```

Comment:
Please, remove the exact common version from the check1. We should get it as a transitive dependency from common library